src/main/webapp/statistics.html

```diff
@@ -231,72 +231,125 @@ <h2>Procedures</h2>
     </script>
     <div id="cost_chart"></div>
     <div id="length_chart"></div>
-    <h2>Abortion Laws</h2>
+    <h2>Laws & Policies</h2>
     <script type="text/javascript">
     google.charts.load('current', {'packages'  : ['geochart'],
     'mapsApiKey': 'AIzaSyBe07-CG9ZcJEpo70rVTDKxsWU-KhFEl9k'
   });
   google.charts.setOnLoadCallback(drawLawMap);
 
   function drawLawMap() {
-    var legalData = google.visualization.arrayToDataTable([
-      ['State' , 'Weeks of legality'],
-      ['Alabama', 6],
-      ['Alaska', 6],
-      ['Arizona', 24],
-      ['Arkansas', 18],
-      ['California', 24],
-      ['Colorado', 24],
-      ['Connecticut', 24],
-      ['Delaware', 20], // *restrictions apply
-      ['Florida', 24],// *restrictions apply
-      ['Georgia', 6],
-      ['Hawaii', 6],
-      ['Idaho', 24],
-      ['Illinois', 6],
-      ['Indiana', 14],
-      ['Iowa', 20],
-      ['Kansas', 6],
-      ['Kentucky', 6],
-      ['Louisiana', 6],
-      ['Maine', 6],
-      ['Maryland', 24],
-      ['Massachusetts', 24],
-      ['Michigan', 20],
-      ['Minnesota', 24],
-      ['Mississippi', 24],
-      ['Missouri', 8],
-      ['Montana', 6],
-      ['Nebraska', 24],
-      ['Nevada', 24],
-      ['New Hampshire', 24],
-      ['New Jersey', 24],
-      ['New Mexico', 24],
-      ['New York', 24],
-      ['North Carolina', 20],
-      ['North Dakota', 12],
-      ['Ohio', 6],
-      ['Oklahoma', 24],
-      ['Oregon', 40],
-      ['Pennsylvania', 24],
-      ['Rhode Island', 24],
-      ['South Carolina', 24],
-      ['South Dakota', 24],
-      ['Tennessee', 6],
-      ['Texas', 20],
-      ['Utah', 28],
-      ['Vermont', 40],
-      ['Virginia', 24],
-      ['Washington', 24],
-      ['West Virginia', 0],
-      ['Wisconsin', 21],
-      ['Wyoming', 6]
+    var legalData = new google.visualization.DataTable();
+    legalData.addColumn('string', 'State');
+    // Use custom HTML content for the domain tooltip.
+    legalData.addColumn('number', 'Legal up to: ');
+    legalData.addColumn({'type': 'string', 'role': 'tooltip', 'p': {'html': true}});
+    legalData.addRows([
+      ['Alabama', 6, createCustomHTMLContent(6, 'https://upload.wikimedia.org/wikipedia/commons/5/5c/Flag_of_Alabama.svg',
```

Comment:
Maybe you can make part of the URL a variable like `var url ="https://upload.wikimedia.org/wikipedia/commons/"` and then fill in the parts that's different for every state